### PR TITLE
Kmohr-soprasteria/issue59 - Update MDIP to use new appPattern release v2.1.2

### PR DESCRIPTION
**Pull request by kmohr-soprasteria**
Update MDIP to use new appPattern release v2.1.2

### CODE REVIEW

File: spec/MultiDomainInventoryProxy.yaml [hunk 1]
```diff
@@ -9333,27 +9333,27 @@ paths:
                       minLength: 3
                       description: >
                         'If body provided, name of RegistryOffice application
-                        find [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-0-http-c-ro-2-0-2-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
+                        find [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-c-ro-2-1-2000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
```
Comment (PrathibaJee) [inline spec/MultiDomainInventoryProxy.yaml:9336]: mdip-1-0-2-http-c-ro-2-1-2000 to be corrected to mdip-1-0-2-http-c-ro-2-1-2-000. 
Similarly , all the following instance in the specification file needs the same correction, 
**RegistryOffice :**
mdip-1-0-2-op-c-bm-ro-2-1-2000, 
mdip-1-0-2-http-c-ro-2-1-2000,
 mdip-1-0-2-tcp-c-ro-2-1-2000
**ExecutionAndTraceLog:**
mdip-1-0-2-http-c-eatl-2-1-2000, 
mdip-1-0-2-op-c-bs-eatl-2-1-2000, 
mdip-1-0-2-tcp-c-eatl-2-1-2000
**Oamlog:**
mdip-1-0-2-http-c-ol-2-1-2000, 
mdip-1-0-2-op-c-bs-ol-2-1-2000, 
mdip-1-0-2-tcp-c-ol-2-1-2000
**AccessAdministrator:**
mdip-1-0-2-http-c-aa-2-1-2000, 
mdip-1-0-2-op-c-bs-aa-2-1-2000, 
mdip-1-0-2-tcp-c-aa-2-1-2000
**ApplicationLayerTopology:**
mdip-1-0-2-http-c-alt-2-1-2000, 
mdip-1-0-2-op-c-bm-alt-2-1-2001, 
mdip-1-0-2-op-c-bm-alt-2-1-2002, 
mdip-1-0-2-op-c-bm-alt-2-1-2003, 
mdip-1-0-2-op-c-bm-alt-2-1-2004, 
mdip-1-0-2-op-c-bm-alt-2-1-2005, 
mdip-1-0-2-tcp-c-alt-2-1-2000

File: spec/MultiDomainInventoryProxy.yaml [hunk 2]
```diff
@@ -12344,6 +12358,551 @@ paths:
...
+                - release-number
+                - protocol
+                - address
+                - port
+              properties:
+                application-name:
+                  type: string
+                  description: >
+                    'Name of the application that is target of the bequeath-your-data-and-die process
+                    update [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-c-ro-1-0-2-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/application-name]'
```
Comment (PrathibaJee) [inline spec/MultiDomainInventoryProxy.yaml:12787]: mdip-1-0-2-http-c-ro-1-0-2-000 should be mdip-1-0-2-http-c-nr-1-0-2-000. Its supposed to point to the new release. in the release-number also the same needs to be modified.

File: spec/MultiDomainInventoryProxy.yaml [hunk 3]
```diff
@@ -12344,6 +12358,551 @@ paths:
...
+                    'Release of the application that is target of the bequeath-your-data-and-die process
+                    update [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-c-ro-1-0-2-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/release-number]'
+                protocol:
+                  type: string
+                  enum:
+                    - 'HTTP'
+                    - 'HTTPS'
+                  description: >
+                    'Protocol to be used during the bequeath-your-data-and-die process
+                    update [/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-tcp-c-ro-1-0-2-000/layer-protocol=0/tcp-client-interface-1-0:tcp-client-interface-pac/tcp-client-interface-configuration/remote-protocol]'
```
Comment (PrathibaJee) [inline spec/MultiDomainInventoryProxy.yaml:12801]: mdip-1-0-2-tcp-c-ro-1-0-2-000 should be changed to mdip-1-0-2-tcp-c-nr-1-0-2-000. Also to the port and protocol.

File: spec/MultiDomainInventoryProxy+config.json [hunk 4]
```diff
@@ -179,26 +179,26 @@
...
             "response-profile-configuration": {
               "value": {
-                "value-reference": "/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-0-http-s-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address"
+                "value-reference": "/core-model-1-4:control-construct/logical-termination-point=mdip-1-0-2-http-s-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address"
               }
             }
           }
         },
         {
-          "uuid": "mdip-1-0-0-response-p-007",
```
Comment (PrathibaJee) [inline spec/MultiDomainInventoryProxy+config.json:188]: For /v1/inform-about-release-history-in-generic-representation , response-profile needs to be included for the all the releases. Currently its for only the 1st release.

**Review by PrathibaJee — CHANGES_REQUESTED** [review]

File: spec/MultiDomainInventoryProxy.yaml (hunk 1 above)
Comment (kmohr-soprasteria) [reply spec/MultiDomainInventoryProxy.yaml:9336]: just noticed that as well, will fix in all specs

**Review by PrathibaJee — APPROVED** [review]

**Review by PrathibaJee — APPROVED** [review]

**Pull request merged**